compiler/circle-interpreter-test/test.lst

```diff
@@ -0,0 +1,26 @@
+#
+# Format:
+#   testcmd(TESTNAME {POS|NEG} COMMAND TO RUN ...)
+#
+#   ex.
+#   testcmd(foo_000 POS echo foo)
+#   testcmd(foo_001 NEG false foo)
+#
+# NOTE: Test can use enviornment variable `WORK_DIR` for model and data
+#       `WORK_DIR` is set as path to `common-artifacts` directory
+#
+
+# Print help message
+testcmd(POS_001 POS -h)
+
+# Invoke circle-interpreter with proper arguments
+testcmd(POS_002 POS ${WORK_DIR}/Conv2D_000.circle ${WORK_DIR}/Conv2D_000.circle.input /tmp/Conv2D_000.output)
```

Comment:
output file will be left in `/tmp` directory. It would be better to remove it.

Reply:
Do you have any great way to do preprocess (which could prepare test materials) and postprocess (such as cleaning by-products)? IMO it can be solved with introducing another macro which removes given files.

Reply:
I don't have "great" idea, but let me share :)

I guess that output file is generated only when (1) circle-interpreter runs with three arguments (2) and returns 0.

So, if you can parse `TESTCMD` here (https://github.com/Samsung/ONE/pull/9658/files#diff-364961265dc34e246ea0cf4c85fdb1e7b6c3a1aafbad633724322cc872d2ad58R45), it will be possible to get the path to the output file. Then run `rm ${OUTPUT_FILE}`.

Something like below

```
if [[ $? -eq 0 ]]; then
  if [[ $TESTTYPE == "POS" ]]; then
    if length of ${TESTCMD} == 3
      OUTPUT_FILE = ${TESTCMD}[2]
      rm ${OUTPUT_FILE}
```

Reply:
Thank you for your idea! :smile: I'll think more how to solve it.

Reply:
FYI, I'm considering to adapt gtest to check the output files' existence and it's validation. I'm looking at `circle-operator-test` script, and will try to make code asap. 🙃